File: src/main/java/kr/nyamnyam/model/domain/UserModel.java
package kr.nyamnyam.model.domain;

import lombok.Data;

@Data
public class UserModel {

    private Long id;
    private String username;
    private String password;
    private String nickname;
    private String name;
    private Long age;
    private String role;
    private String tel;
    private String gender;
    private Boolean enabled;


}

File: src/app/model/user.model.ts
export interface User {
    id: string; // MongoDB의 ID는 보통 문자열입니다.
    username: string;
    password: string;
    nickname: string;
    name: string;
    age: number;
    role: string;
    tel: string;
    gender: string;
    enabled: boolean;
    imgId: number | null; // imgId가 없을 수도 있으므로 null로 설정
}
